pkg/taskrunmetrics/metrics_test.go

```diff
@@ -286,7 +363,7 @@ func TestRecordTaskRunDurationCount(t *testing.T) {
 				t.Fatalf("NewRecorder: %v", err)
 			}
 
-			if err := metrics.DurationAndCount(c.taskRun); err != nil {
+			if err := metrics.DurationAndCount(c.taskRun, nil); err != nil {
```

Comment:
this should have the `beforeCondition` instead of `nil` 🙃 

```
if err := metrics.DurationAndCount(c.taskRun, c.beforeCondition); err != nil {
```

Reply:
Yes, that's why test was passing. I fixed this.